src/pages/data-reporting/changelog/index.mdx

```diff
@@ -0,0 +1,24 @@
+---
```

Comment:
@jakubsikora suggested using `git mv` instead of simply moving files. Because of that, we can keep the history of files.

Reply:
Yep exactly